$Id: news_dont_break_feeds.patch 344 2012-06-29 15:38:56Z tg $

Unbreak RSS links generated by other things than the News extension
itself, such as those on the &action=history pages. The cost of this
is that the newly introduced (by this extension) <newsfeed> tag does
no longer work, but that’s no regression…

--- a/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/News/News.php
+++ b/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/News/News.php
@@ -32,7 +32,11 @@ $wgNewsFeedUserPattern = false; // patte
 $wgAutoloadClasses['NewsRenderer'] = dirname( __FILE__ ) . '/NewsRenderer.php';
 $wgAutoloadClasses['NewsFeedPage'] = dirname( __FILE__ ) . '/NewsRenderer.php';
 $wgHooks['ArticleFromTitle'][] = 'wfNewsArticleFromTitle';
-$wgHooks['SkinTemplateOutputPageBeforeExec'][] = 'wfNewsSkinTemplateOutputPageBeforeExec';
+/*-
+ * disabled to not break page feeds from other sources
+ * cf. https://evolvis.org/tracker/t_follow.php/2557
+ */
+//$wgHooks['SkinTemplateOutputPageBeforeExec'][] = 'wfNewsSkinTemplateOutputPageBeforeExec';
 $wgHooks['ParserFirstCallInit'][] = 'wfNewsSetHooks';
 
 //FIXME: find a way to override the feed URLs generated by OutputPage::getHeadLinks
